server/middlewares/uploadLogoToCloud.js

```diff
@@ -0,0 +1,31 @@
+import rimraf from 'rimraf';
+import dotenv from 'dotenv';
+
+dotenv.load();
+const cloudinary = require('cloudinary').v2;
+
+const uploadLogoToCloud = (req, res, next) => {
+  // console.log('jjjjjjjjjjjjjjjj');
```

Comment:
Get rid of console log and commented code

Reply:
This middleware has been merged with the uploadLogo middleware and the commented code deleted. 
As discussed on Friday, the .env.sample has been added but not on this branch. 
I added it together with some other configuration changes on [another branch](https://github.com/haywhyze/politico/commit/9b4b5aba5941140330b4e1766ec06ff1358382c1) to deploy the app to heroku to aid remote access to the app. 
I'll raise a PR for that as soon as this has been merged. 
Most recent PR feature ready for review will be live [here](https://politico-yusuf.herokuapp.com/api/v1/) so it can be tested on postman without having to clone and add api keys.